src/BridgedRoom.ts

```diff
@@ -676,12 +676,36 @@ export class BridgedRoom {
             return;
         }
 
+        let sendAsLink = false;
+        const authToken = this.SlackClient?.token;
```

Comment:
Did you mean to use the parameter `client` or why did you add that above?

Reply:
I think the param was used originally then removed. Removing it now.